Makefile

```diff
@@ -23,6 +23,17 @@ go:
 	@echo "Generating go protobuf files"
 	docker run --pull always --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -d protos -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
 
+python: env/pyenv.cfg
+	@echo "Generating python protobuf files"
+	. env/bin/activate && cd ./python/sigstore_protobuf_specs && python -m grpc_tools.protoc -I ../../protos/ --python_betterproto_out=. ../../protos/*
+
+env/pyenv.cfg:
+	@echo "Building virtual environment for python protobuf"
+	python3 -m venv env
+	./env/bin/python -m pip install --upgrade pip
+	./env/bin/python -m pip install -e ./python[dev]
+	./env/bin/python -m pip install grpcio-tools
```

Comment:
At the moment, I'm installing `grpcio-tools` to invoke the Protobuf compiler.

Before we merge this, I can either fiddle with this to plug `betterproto` into the Docker image's compiler or we can remove the Docker container altogether and make everything use this one.

Reply:
I'm personally okay with that, although IMO the `grpcio-tools` dependency should go in the `[dev]` extra rather than being a standalone dependency here.

Reply:
Since we'll be moving to executing this within Docker, I think I'll just use the `protoc` in the image.

Reply:
Sounds good!